samples/bluetooth/periodic_adv_rsp/src/main.c

```diff
@@ -369,6 +366,12 @@ int main(void)
 		printk("PAwR config written to sync %d, disconnecting\n", num_synced - 1);
 
 disconnect:
+		/* Adding delay (2ms * interval value, using 2ms intead of the 1.25ms
+		 * used by controller) to ensure sync is established before
+		 * disconnection.
+		 */
+		k_sleep(K_MSEC(per_adv_params.interval_max * 2));
```

Comment:
I guess this is ok. The samples do not handle failure well. A better solution might be for the sync sample to do the disconnect once it has synced to the pawr train, but that is not needed for this PR.